package.json

```diff
@@ -5,12 +5,13 @@
   "scripts": {
     "banners": "node tools/maintain-banners.js {okta-hosted-login,custom-login,tools,scripts}/{*js,src/**/*{js,jsx,css}}",
     "lint": "npx eslint -c .eslintrc.json --ext .js --ext .jsx okta-hosted-login/src/ custom-login/src/ scripts/",
-    "test": "npm run lint && npm run test:e2e",
+    "test": "npm run lint && npm run test:unit && npm run test:e2e",
     "okta-hosted-login-server": "npm start --prefix okta-hosted-login/",
     "test:okta-hosted-login": "export TEST_TYPE=implicit || setx TEST_TYPE \"implicit\" && protractor okta-oidc-tck/e2e-tests/okta-hosted-login/conf.js",
     "custom-login-server": "npm start --prefix custom-login/",
     "test:custom-login": "export TEST_TYPE=implicit || setx TEST_TYPE \"implicit\" && protractor okta-oidc-tck/e2e-tests/custom-login/conf.js",
     "resource-server": "node scripts/node-resource-server.js",
+    "test:unit": "npm run test --prefix okta-hosted-login/ && npm run test --prefix custom-login/",
```

Comment:
I'm not 100% sold on having the root level use `npm run test:unit` while the individual packages use `npm run test`, but I'm not 100% against it either.

Reply:
how about use `test:unit` in both places for consistency?

Reply:
After checking the code generated from `create-react-app`, I think it might make sense to keep the current naming conventions. As `test` in each individual package matches the script generated from `CRA`. And the root level `test:unit` is for the CI usage.